manifests/h/Helix/Helix/22.08.1/Helix.Helix.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Helix.Helix
+PackageVersion: 22.08.1
+InstallerType: zip
+Commands:
+- hx
+ReleaseDate: 2022-09-01
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The package installs, but requires the dependency to run:
![image](https://user-images.githubusercontent.com/114952565/217892982-384781d0-bc5e-4e45-a3ed-99f0b129ed63.png)